client/package.json

```diff
@@ -19,6 +19,7 @@
     "preset": "jest-expo"
   },
   "dependencies": {
+    "babel-preset-expo": "^4.0.0",
```

Comment:
Do we need this on top of "babel-present-env". What is this for?

Reply:
errrr no idea lol i think it asked me to download it